test/oauth2_session_test.py

```diff
@@ -5,6 +5,12 @@
 
 
 TEST_DOMAIN = 'test.getbynder.com'
+VERSION = None
+
+
+with open('VERSION') as fh:
+    VERSION = fh.read().strip()
+    fh.close()
```

Comment:
```suggestion
VERSION = 'whatever'
```
No need to read it from a file I'd say, this would only make the tests slower and is not relevant for what we're testing.